runtime-modules/content/src/lib.rs

```diff
@@ -1360,6 +1474,12 @@ impl<T: Trait> Module<T> {
         }
     }
 
+    fn bag_id_for_channel(channel_id: &T::ChannelId) -> storage::BagId<T> {
+        // retrieve bag id from channel id
+        let dyn_bag = DynamicBagIdType::<T::MemberId, T::ChannelId>::Channel(channel_id.clone());
+        BagIdType::<T::MemberId, T::ChannelId>::Dynamic(dyn_bag)
```

Comment:
`From` trait is implemented for this conversion.

Reply:
done